tests/texlive/latexdiff.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test the latexdiff command with a simple example
+# Maintainer: Katerina Lorenzova <klorenzova@suse.cz>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console('root-console');
+    zypper_call('in texlive-latexdiff-bin', timeout => 1800);
+    select_console('x11');
+    x11_start_program('xterm');
+
+    assert_script_run 'mkdir texlive';
+    assert_script_run 'cd texlive';
+    assert_script_run 'wget ' . data_url('texlive/original.tex');
+    assert_script_run 'wget ' . data_url('texlive/modify.tex');
```

Comment:
There should not be a need to download the data files as they should have been extracted to a local directory already, e.g. see http://10.100.51.129/tests/43#step/prepare_test_data/21